.github/workflows/autoprerelease.yml

```
@@ -0,0 +1,42 @@
+name: generate changelog and create pre release
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)